### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by grikomsn**
This PR polishes up the readme which initially attempt to fix a typo. Merge is optional. ✌🏻

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,44 +1,61 @@
 # remark-smartypants
 
+[![v](https://badgen.net/npm/v/remark-smartypants)][npm]
+[![dt](https://badgen.net/npm/dt/remark-smartypants)][npm]
```
Comment (silvenon) [inline readme.md]: Not sure if it makes sense to add this text considering that the most important piece of information is missing (which version and how many downloads). Alternatively leave the alt text out completely, your decision.

```suggestion
[![package version](https://badgen.net/npm/v/remark-smartypants)][npm]
[![number of downloads](https://badgen.net/npm/dt/remark-smartypants)][npm]
```

File: readme.md [hunk 2]
```diff
@@ -1,44 +1,61 @@
...
-import mdx from '@mdx-js/mdx'
-import smartypants from 'remark-smartypants'
+import mdx from "@mdx-js/mdx";
+import smartypants from "remark-smartypants";
 
-const result = await mdx('# <<Hello World!>>', {
+const result = await mdx("# <<Hello World!>>", {
   remarkPlugins: [
     smartypants,
+    //
```
Comment (silvenon) [inline readme.md]: Did you mean to put something here?

File: readme.md [hunk 3]
```diff
@@ -1,44 +1,61 @@
...
 
+## Usage
+
+Example using [remark]:
+
 ```js
-import remark from 'remark'
-import smartypants from 'remakr-smartypants'
+import remark from "remark";
+import smartypants from "remark-smartypants";
```
Comment (silvenon) [inline readme.md]: It seems like you have a Prettier extension that's formatting this, could you either turn it off or set it to single quotes and no semicolons?

**Review by silvenon — CHANGES_REQUESTED** [review]
Thank you very much! I have minor suggestions and questions. 🙂

**Comment by grikomsn** [comment]
Force pushed with only needed changes. Apologies for the unnecessary commits.

**Comment by silvenon** [comment]
No problem, thank you so much!

**Review by silvenon — APPROVED** [review]

**Pull request merged**